AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1555,7 +1555,7 @@ iembra2or.com,battle-one.com,terbit21.online,wizardsubs.com,audio-sound-premium.
 ! #%#//scriptlet("remove-attr", "oncontextmenu|onselectstart|ondragstart", "body")
 novelpia.com,dragonball-tube.com,onepiece-tube.com,fairytail-tube.org,baho.com.tr,wholehk.com,siyasetcafe.com,webtv.dothome.co.kr,outdoornews.co.kr,egloos.com,novel.naver.com,eskisehirekspres.net#%#//scriptlet("remove-attr", "oncontextmenu|onselectstart|ondragstart", "body")
 !
-m.webtoons.com#%#AG_onLoad(function() { var j = document.querySelectorAll('img._checkVisible'); for(var i = 0; i < j.length; i++) { j[i].oncontextmenu = j[i].onselectstart = j[i].ondragstart = null; }})
+m.webtoons.com#%#AG_onLoad(function() { const ObserverFunc = function() { if( document.querySelector('body').classList[3] == undefined ) { var j = document.querySelectorAll('img._checkVisible'); for(var i = 0; i < j.length; i++) { j[i].oncontextmenu = j[i].onselectstart = j[i].ondragstart = null; }}}; const Observer = new MutationObserver(ObserverFunc); Observer.observe(document.body, {attributes: true}); })
```

Comment:
This also should work fine
```
m.webtoons.com#%#AG_onLoad(function() {var g = new MutationObserver(function() {var c = document.querySelectorAll('img._checkVisible');for (var i = 0; i < c.length; i++) {c[i].oncontextmenu = c[i].onselectstart = c[i].ondragstart = null;}});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)});
```

Reply:
Okay. Your change works on my environment.